.github/workflows/nemo_tests.yml

```diff
@@ -101,6 +101,18 @@ jobs:
         tail -n 1 output.txt | grep -q "S_max:  0.407622"
         grep -A 1 "Elapsed Time" output.txt
         echo  $GITHUB_REF_NAME $GITHUB_SHA $(grep -A 1 "Elapsed Time" output.txt | head -n 2 | tail -n 1) >> ${NEMO_DIR}/performance_history_openmp_gpu
+        # Upload the results into a Github Gist
```

Comment:
Share a Python script with the lfric_test.yml?

Reply:
Using the suggested github action now